app/src/androidTest/java/com/ustwo/boilerplate/application/ApplicationTestRunner.java

```diff
@@ -0,0 +1,15 @@
+package com.ustwo.boilerplate.application;
+
+import android.app.Application;
+import android.content.Context;
+import android.support.test.runner.AndroidJUnitRunner;
+
+public class ApplicationTestRunner extends AndroidJUnitRunner {
+
+  @Override
+  public Application newApplication(final ClassLoader cl, final String className, final Context
```

Comment:
Is it possible not to abbreviate this? It's not something commonly done elsewhere in this codebase.